config-model/src/main/javacc/IntermediateParser.jj

```diff
@@ -1910,7 +1907,12 @@ void function(ParsedRankProfile profile) :
     {
         func.setExpression(expression);
         func.setInline(inline);
-        profile.addFunction(func);
+        var old = profile.addOrReplaceFunction(func);
+        if (old != null) {
+            deployLogger.logApplicationPackage(Level.WARNING, "Function '" + func.name()
+                                               + "' replaces a previous function with the same name in rank profile '"
```

Comment:
Shouldn't this just say "[function] is defined twice"?

Reply:
> Shouldn't this just say "[function] is defined twice"?

I will change this (the unit test was very strict)